src/main.ts

```diff
@@ -205,11 +219,15 @@ async function getActionFile(
       path: 'action.yml'
     })
 
+ 
+
```

Comment:
Let's not add extra empty lines, please remove

Reply:
removed extra lines